README.md

```diff
@@ -4,7 +4,7 @@ This repository generates the main docker-compose.yml (follow the steps to gener
 
 The `docker-compose.yml` file is pushed to [Balena](https://www.balena.io/) (using GitHub Actions), which in turn pulls down the various Docker images outlined below.
 
-There are currently five different services running within this device, which are all outlined below.
+There are currently four different services running within this device, which are all outlined below.
```

Comment:
five is right now as we have the multiplexer too

Reply:
Actually it is six right now, but I guess `dbus` is not counted? :-)